.travis.yml

```diff
@@ -14,24 +15,39 @@ cache:
     - $HOME/.composer/cache
 
 env:
-  - SYMFONY_VERSION=2.5.*
+  - SYMFONY_VERSION=2.6.*
 
 matrix:
   include:
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.3.*
-    - php: 5.5
-      env: SYMFONY_VERSION=2.4.*
-    - php: 5.5
-      env: SYMFONY_VERSION=2.6.*
-
-before_script:
+    - php: 5.6
+      env: SYMFONY_VERSION=2.7.*
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*
+    - php: 5.6
+      env: SYMFONY_VERSION=3.0.*
+    - php: 5.3
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
+  allow_failures:
+    - php: nightly
```

Comment:
we need migrate away from the use of ``String`` annotation to be able to pass the tests with php7